packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -217,8 +220,15 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
           component = this.getComponentForRendering(componentRendering);
         }
 
+        // Fallback/defaults for Sitecore Component renderings (in case not defined in component factory)
         if (componentRendering.componentName === FEAAS_COMPONENT_RENDERING_NAME) {
           component = FEaaSComponent;
+        } else if (componentRendering.componentName === FEAAS_WRAPPER_RENDERING_NAME) {
```

Comment:
And to follow up on the plugin comment: do we need both the feaas plugin and fallbacks here?

Reply:
It should be here in case component factory doesn't have new components added, for existing customers it's a good solution

Reply:
Correct. Though one thing they _wouldn't_ get with this fallback solution is Next.js SSR for `FEaaSWrapper` (since `getStatic/ServerSideProps` is only in the `sitecore-jss-nextjs` package re-export of the component). So it would still be in their best interest to add the plugin to their solution.